manifests/service.pp

```diff
@@ -6,7 +6,7 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  service { 'jenkins':
+  service { $::jenkins::params::service_name:
```

Comment:
What is the reason for this change?  It seems unrelated to the feature being proposed in this PR.